### PR TITLE
fix(table): http example needs httpclientmodule

### LINKED ISSUE #339: HttpClientModule is missing in stackblitz MatTableModule examples
When you open stackblitz's example in the [example's tab](https://material.angular.io/components/table/examples) => "Table retrieving data through HTTP" example, `HttpClientModule` is missing, so the example doesn't run.

image

### PR DESCRIPTION
**Pull request by andrewseguin**
Current example is broken since it requires the new Http module.

Fixes #339

### CODE REVIEW

**Comment by mmalerba** [comment]
is there a corresponding fix in material2 repo that this is generated from?

**Comment by andrewseguin** [comment]
Yeah this one here - https://github.com/angular/material2/commit/0ea43700efb105aa02ad3b1d3f800bf4a9925883#diff-c3b988af35d5ab4790e4795e2d7015dd

File: src/assets/stackblitz/main.ts [hunk 1]
```diff
@@ -86,6 +87,7 @@ export class DemoMaterialModule {}
     BrowserAnimationsModule,
     FormsModule,
     HttpModule,
```
Comment (Splaktar) [inline src/assets/stackblitz/main.ts]: Should we remove `HttpModule`?

Comment (josephperrott) [reply src/assets/stackblitz/main.ts]: `HttpModule` will be removed everywhere in a different PR.

Comment (andrewseguin) [reply src/assets/stackblitz/main.ts]: Good call - let's cleanup in another PR. Would like to at least just make sure this fix goes in without the risk of breaking other examples if I miss a case of one using `HttpModule`

**Pull request merged**